src/yunohost/vendor/acme_tiny/acme_tiny.py

```diff
@@ -136,12 +136,13 @@ def _poll_until_not(url, pending_statuses, err_msg):
             wellknown_file.write(keyauthorization)
 
         # check that the file is in place
-        try:
-            wellknown_url = "http://{0}/.well-known/acme-challenge/{1}".format(domain, token)
-            assert(disable_check or _do_request(wellknown_url)[0] == keyauthorization)
-        except (AssertionError, ValueError) as e:
-            os.remove(wellknown_path)
-            raise ValueError("Wrote file to {0}, but couldn't download {1}: {2}".format(wellknown_path, wellknown_url, e))
+        if not no_checks:
+            try:
+                wellknown_url = "http://{0}/.well-known/acme-challenge/{1}".format(domain, token)
+                assert(disable_check or _do_request(wellknown_url)[0] == keyauthorization)
```

Comment:
`disable_check` is used here so we might want to change the API inside YunoHost instead, no?

Reply:
Ah indeed xD Didnt event took the time to read the new args